modules/publisher/views.py

```diff
@@ -319,56 +319,60 @@ def build_image_info(image, image_type):
 
 
 def post_market_snap(snap_name):
-    snap_id = api.get_snap_id(snap_name, flask.session)
+    changes = loads(flask.request.form['changes'])
```

Comment:
I did a little hack :)
- In console enable Apply button, and remove `event.preventDefault();`
- Submit
- This causes a but because `flask.request.form['changes']` doesn't exist

Reply:
Thanks, fixed